doc/source/user_guide/timeseries.rst

```diff
@@ -3,7 +3,7 @@
 {{ header }}
 
 ********************************
-Time series / date functionality
+Time Series / date functionality
```

Comment:
Since it is not a single word, is there a specific reason to capitalize Series?

Reply:
Oh I see. I'll correct that.